src/gui/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ include(ECMAddAppIcon)
 
 find_package(Qt5 REQUIRED COMPONENTS Widgets)
 
+#if (WIN32)
```

Comment:
If this is an optional dependency, consider introducing a CMake variable allowing users to opt into adding it. Could also be used in the code to opt into setting that icon.

Reply:
I don't consider it optional.
Optional dependencies never solved an issue, optional Qt modules certainly never did that.